all/guide/passthrough.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebarDepth: 2
+title: Passthrough migration
+---
+
+# Passthrough migration
+
+::::: row
+:::: left
+
+This guide describes the **passthrough** strategy for migrating password-based login from your existing system to Userfront.
+
+Using this approach, you get a standard set of client-side interactions for all new and existing users while you migrate your login to a new system.
+
+::::
+:::: right
+
+::: tip
+Userfront does not store user passwords in a recoverable format (it stores a [hash](https://en.wikipedia.org/wiki/Cryptographic_hash_function)), and this guide assumes that your system also does not store passwords in a recoverable format.
```

Comment:
Would this customer want it explicitly stated that the hash is salted, etc.?

Reply:
We'll have more info on this on the upcoming security page, and can go through and add links then.